tests/e2e/unified_test.go

```diff
@@ -81,7 +83,33 @@ func TestAllInSeries(t *testing.T) {
 		}
 	})
 
-	t.Run("fixtures", func(t *testing.T) {
+	testFixturesInSeries(ctx, t, "AllInSeries", false, cancel)
+}
+
+// TestPauseInSeries is a basic smoke test to prove that if CC pauses actuation of resources
+// via the actuationMode field, then resources are not actuated onto the cloud provider.
+// The current test is to make sure that POST requests are not recorded as HTTP events.
+func TestPauseInSeries(t *testing.T) {
+	ctx := context.Background()
+	ctx, cancel := context.WithCancel(ctx)
+	t.Cleanup(func() {
+		cancel()
+	})
+
+	testFixturesInSeries(ctx, t, "PauseInSeries", true, cancel)
+}
+
+func testFixturesInSeries(ctx context.Context, t *testing.T, testName string, testPause bool, cancel context.CancelFunc) {
+	t.Helper()
+
+	if os.Getenv("RUN_E2E") == "" {
+		t.Skip("RUN_E2E not set; skipping")
+	}
+	if testPause && os.Getenv("GOLDEN_REQUEST_CHECKS") == "" {
+		t.Skip("GOLDEN_REQUEST_CHECKS not set; skipping as this test relies on the golden files.")
```

Comment:
Fine for now, but maybe we should always record the HTTP traffic, and use GOLDEN_REQUEST_CHECKS only to control if we do the CompareGoldenFile call.

Reply:
I think that may require some refactoring below but I can do so in a follow up -- want to focus more on the main PR for pause for now.